lib/take_away.rb

```diff
@@ -0,0 +1,21 @@
+ require_relative 'menu'
+class TakeAway
+
+attr_reader :menu, :order
+
+  def initialize(menu = Menu.new, order = nil)
+    @menu = menu
+    @order = order
+  end
+
+  def print_menu
+    menu.print_dishes
+  end
+  
+  # def order_items(dishes)
+  #   dishes.each |item, quantity|
```

Comment:
I think here you want to access each dish with the each and then access the item and quantity from the hash at each index and do the order.add